raysort/sort_utils.py

```diff
@@ -508,7 +512,19 @@ def split_part(part, pivot) -> Tuple[np.ndarray, np.ndarray]:
     blocks = sortlib.sort_and_partition(copy_of_part, [0, pivot])
 
     split_idx, _ = blocks[1]
-    part_one = part[0:split_idx]
-    part_two = part[split_idx:]
-
-    return part_one, part_two
+    return part[:split_idx], part[split_idx:]
+
+
+@ray.remote(num_cpus=1)
+def make_chunks(part: np.ndarray) -> list[np.ndarray]:
+    """converts a part array into chunks of 100 MB"""
```

Comment:
change documentation to this:

Splits an array in the object store into 100MB chunks. Returns a list of references to the chunks.

Also, the return type should be `list[ray.ObjectRef]`, yes?